bin/pd-nagios

```diff
@@ -176,6 +185,18 @@ def parse_fields(fields):
         return {}
     return dict(f.split("=", 1) for f in fields)
 
+def parse_env_fields(details, env_fields):
+    if env_fields is None:
+        return
+    import os
+    for f in env_fields:
+        v = os.environ.get("NAGIOS_%s" % f)
+        if v is None:
+            v = os.environ.get("ICINGA_%s" % f)
+        if v is None:
```

Comment:
Could do. Though I'm not sure you'd get a blank key?  I was leaning towards always sending the field even if the value is blank so that the user knows there's no bug in the pd parts of the stack.